app/components/app_child_summary_component.rb

```diff
@@ -32,6 +34,12 @@ def call
             href:,
             visually_hidden_text: "NHS number"
           )
+        elsif (href = pds_search_history_link)
+          row.with_action(
+            text: "PDS history",
+            href:,
+            visually_hidden_text: "PDS search history"
```

Comment:
I _think_ this text gets appended to the visible text, meaning this link would be read aloud as ‘PDS history PDS search history’. Probably best to not show any visually hidden text for this link,